app/views/groups/splash.html.erb

```diff
@@ -1,14 +1,14 @@
-<section class="section-3">
-  <div class="header-wrapper">
-    <h1 class="title">
-      Budget Tracker
+<section class="">
+  <div class="splash_container">
+    <h1 class="splash_name">
+      Budget App
     </h1>
-        <p class="description">
-      A simple app to track your budget.
+        <p class="splash_p">
+      A user-friendly app to help you track your expenses.
     </p>
-    <div class="button-wrapper">
-      <%= button_to "SIGN IN", new_user_session_path, method: :get, class: '' %>
-      <%= button_to "SIGN UP", new_user_registration_path, method: :get, class: '' %>
     </div>
+    <div class="">
+      <%= button_to "SIGN IN", new_user_session_path, method: :get, class: 'login_splash' %>
+      <%= button_to "SIGN UP", new_user_registration_path, method: :get, class: 'logout_splash' %>
```

Comment:
- [ ] Kindl make sure this page looks like the orignal design, the text should be desolay the same way it is in the tempalte. and use the same font-family and font-size

youre design | the template
---|---
<img width="210" alt="image" src="https://github.com/chepkok3/Budget-app/assets/65068771/eeef3027-7a6f-42e6-9b31-28d1cbcc43d7"> | <img width="210" alt="image" src="https://github.com/chepkok3/Budget-app/assets/65068771/ece00b71-081f-4cf0-bb5c-2fd34b23596e">